pychiquito/dsl.py

```diff
@@ -97,15 +97,12 @@ def gen_witness(self: Circuit, args: Any) -> TraceWitness:
     def get_ast_json(self: Circuit) -> str:
         return json.dumps(self.ast, cls=CustomEncoder, indent=4)
 
-    def ast_to_halo2(self: Circuit):
```

Comment:
@qwang98 I never meant for this method to be removed, just that the user should not use it, because halo2_mock_prover will already call it. Anyway, it does not matter much